internal/rest/resources/resources.go

```diff
@@ -92,49 +83,66 @@ func ValidateEndpoints(extensionServers map[string]rest.Server, coreAddress stri
 			serverAddresses[server.Address.String()] = true
 		}
 
-		// Ensure no endpoint path conflicts with another endpoint on the same server.
-		// If a server lacks an address, we need to compare it to every other server
-		// that also lacks an address, as well as the internal endpoints.
-		serverPaths, err := resourcesConflict(server, existingEndpointPaths)
+		existingPaths := extensionPaths[serverName]
+		if server.CoreAPI {
+			existingPaths = corePaths
+		}
+
+		err := resourcesConflict(server, existingPaths)
 		if err != nil {
 			return err
 		}
```

Comment:
Hmm, right now I think this would always return an error for all valid Servers. 

`resourcesConflict` doesn't make sense anymore because it's checking for existence in the map and erroring out, but the map it's supplied with is the set of all paths that exist on the server, so it will always error for that server.